apps/src/netsim/NetSimApi.js

```diff
@@ -146,6 +147,26 @@ var tableApi = {
   },
 
   /**
+   * Insert a row into the table.
+   * @param {Object} value - desired row contents, must be JSON.stringify-able.
+   * @param {NodeStyleCallback} callback - Expected result is the created
+   *        row object (which will include an assigned 'id' key).
+   */
+  createRow: function(value, callback) {
+    this.post_(JSON.stringify(value), callback);
+  },
+
+  /**
+   * Insert several rows into the table.
+   * @param {Object[]} values - desired row contents, must be JSON.stringify-able.
+   * @param {NodeStyleCallback} callback - Expected result is an array of the created
+   *        row object (which will include assigned 'id' keys).
+   */
+  createRows: function(values, callback) {
+    this.post_(JSON.stringify(values), callback);
+  },
```

Comment:
It feels a little funny to see these two functionally identical but semantically different functions.  I'd be tempted to say you should just expose one, and make the type of the first param {Object|Object[]}.